fiftyone/core/dataset.py

```
@@ -8307,6 +8307,33 @@ def _make_sample(self, d):
     def _sample_dict_to_doc(self, d, *, _reload_backing_docs=True):
```

Ruff (0.14.10)

Missing return type annotation for private function `_sample_dict_to_doc`

(ANN202)